examples/server_hardware.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+from pprint import pprint
+from hpOneView.oneview_client import OneViewClient
+from hpOneView.exceptions import HPOneViewException
+from config_loader import try_load_from_file
+
+config = {
+    "ip": "172.16.102.59",
+    "credentials": {
+        "authLoginDomain": "",
+        "userName": "administrator",
+        "password": ""
+    }
+}
+
+# Try load config from a file (if there is a config file)
+config = try_load_from_file(config)
+
+oneview_client = OneViewClient(config)
+
+# Get Statistics with defaults
+print("Get server-hardware statistics")
```

Comment:
Perhaps, but so we should change all other examples too, they are similar. What do you think?